rhel7/profiles/stig-rhel7-disa.profile

```diff
@@ -17,6 +17,7 @@ description: "This profile contains configuration checks that align to the \n
 selections:
     - login_banner_text=dod_banners
     - inactivity_timeout_value=15_minutes
+    - var_sssd_ssh_known_hosts_timeout=5_minutes
```

Comment:
This rule and variable can actually be removed from the profile as it appears to no longer exist.

Reply:
I'm not sure I follow this point -- isn't that variable added in this PR?

Reply:
If you check the stig references, the sssd_ssh_known_hosts_timeout rule no longer exists. So, we can remove it from the stig profile. So, add that here or a new PR?

Reply:
Ah sorry, I missed the "rule" part -- I only read the "variable" part and was wondering why you thought the variable was removed from the profile. :)